.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
```

Comment:
I don't know about yml file. Is this file template?

Reply:
Yeah, it's a kind of markup language and used by some frameworks.
If we locate this file, CodeQL scanning for PRs will start to wrok.
<img width="857" alt="image" src="https://user-images.githubusercontent.com/5474642/181156226-1ec3c93c-027b-4b3d-8225-bc55eb83cc65.png">

Reply:
And as for the contents of this file, yes it's a template provided by Github.

Reply:
Got it!